### PR TITLE
add checks for type error

### PR DESCRIPTION
**Pull request by Danielecina**
#### Checklist

- [x] your branch will not cause merge conflict with `master`
- [x] run `npm test`
- [ ] tests are included
- [ ] the documentation is updated or integrated accordingly with your changes
- [x] the code will follow the lint rules and style of the project
- [x] you are not committing extraneous files or sensible data

### CODE REVIEW

File: packages/api-explorer/src/JsonForm/arrays-editor.js [hunk 1]
```diff
@@ -11,6 +11,7 @@ module.exports = (classReference) => class arrayCustomEditor extends classRefere
       if (
         this.jsoneditor &&
         this.jsoneditor.schema &&
+        this.jsoneditor.schema.required &&
         this.jsoneditor.schema.required.includes(this.key) &&
         Array.isArray(this.jsoneditor.schema.required)
```
Comment (davidebianchi) [inline packages/api-explorer/src/JsonForm/arrays-editor.js:16]:
```suggestion
        Array.isArray(this.jsoneditor.schema.required) &&
        this.jsoneditor.schema.required.includes(this.key) &&
```
In this way could be better?

Comment (Danielecina) [reply packages/api-explorer/src/JsonForm/arrays-editor.js]: better

**Comment by fredmaggiowski** [comment]
LGTM

**Review by davidebianchi — APPROVED** [review]
LGTM

**Pull request merged**